src/components/ZclCommandManager.vue

```diff
@@ -50,6 +50,38 @@ limitations under the License.
             >
               {{ validationErrorMessage }}
             </q-tooltip>
+            <!-- <q-icon
```

Comment:
not related to Vue Tour.
Is this being accidentally added due to merge/rebase?

Reply:
some of these commented code came from some of rebase/merge I should check the code again I think